.codecov.yml

```diff
@@ -2,11 +2,6 @@
 coverage:
   precision: 1
   round: down
-
-coverage:
   status:
     patch:
       default: false
-
-# https://docs.codecov.io/docs/pull-request-comments#section-requiring-changes
-comment: off
```

Comment:
@ecomba was the removal intentional? Has the tool changed? In the past unless explicitly disabled, it was commenting every PR.

Reply:
> @ecomba was the removal intentional? Has the tool changed? In the past unless explicitly disabled, it was commenting every PR.

No, this wasn't intentional. What I think happened is that I might have run the tool and it might have replaced the content of the file? I do not recollect changing this file. Will add the deleted lines...